lib/cloudregister/registerutils.py

```diff
@@ -1274,6 +1274,50 @@ def update_ca_chain(cmd_w_args_lst):
     return 0
 
 
+# ----------------------------------------------------------------------------
+def update_rmt_certs():
+    """Check if the update server certs have changed and if yes update the
+       system accordingly"""
+    # We are currently getting the latest cert information, nothing to do
+    if is_new_registration():
+        return
+    cached_rmt_servers = get_available_smt_servers()
+    proxy = set_proxy()
+    proxies = None
+    if proxy:
```

Comment:
variable `proxy` can be removed and use
```python
if set_proxy():
```
then the name of the method could be discussed as well, but that's outside the review